grpc/src/main/java/bisq/grpc/BisqGrpcClient.java

```diff
@@ -0,0 +1,298 @@
+                                .map(bisq.core.offer.Offer::fromProto)
+                                .collect(Collectors.toList());
+                        result = offers.toString();
+                        break;
+                    case "getPaymentAccounts":
+                        List<PaymentAccount> paymentAccounts = getPaymentAccounts().stream()
+                                .map(proto -> PaymentAccount.fromProto(proto, corePersistenceProtoResolver))
+                                .collect(Collectors.toList());
+                        result = paymentAccounts.toString();
+                        break;
+                    case "placeOffer":
+                        // test input: placeOffer CNY BUY 750000000 true -0.2251 1000000 500000 0.12 5a972121-c30a-4b0e-b519-b17b63795d16
+                        // payment accountId and currency need to be adopted
+
+                        // We expect 9 params
+                        // TODO add basic input validation
+                        try {
+                            checkArgument(params.size() == 9);
+                            String currencyCode = params.get(0);
+                            String directionAsString = params.get(1);
+                            long priceAsLong = Long.parseLong(params.get(2));
+                            boolean useMarketBasedPrice = Boolean.parseBoolean(params.get(3));
+                            double marketPriceMargin = Double.parseDouble(params.get(4));
+                            long amountAsLong = Long.parseLong(params.get(5));
+                            long minAmountAsLong = Long.parseLong(params.get(6));
+                            double buyerSecurityDeposit = Double.parseDouble(params.get(7));
+                            String paymentAccountId = params.get(8);
+                            boolean success = placeOffer(currencyCode,
+                                    directionAsString,
+                                    priceAsLong,
+                                    useMarketBasedPrice,
+                                    marketPriceMargin,
+                                    amountAsLong,
+                                    minAmountAsLong,
+                                    buyerSecurityDeposit,
+                                    paymentAccountId);
+                            result = String.valueOf(success);
+                            break;
+                        } catch (Throwable t) {
+                            log.error(t.toString(), t);
+                            break;
+                        }
+                    case "stop":
+                        result = "Shut down client";
+                        try {
+                            shutdown();
+                        } catch (InterruptedException e) {
+                            log.error(e.toString(), e);
+                        }
+                        break;
+                    case "stopServer":
```

Comment:
Somehow this command didn't work for me.

Reply:
I just checked it locally and it does work for me. That is, after I call the `stopServer` RPC from the client, no further commands can be issued from the client because indeed the server has been stopped.